packages/__docs__/src/index.js

```diff
@@ -29,6 +29,11 @@ import { App } from './App'
 import { canvas } from '@instructure/ui-themes'
 import { InstUISettingsProvider } from '@instructure/emotion'
 import '../globals'
+import { Buffer } from 'buffer'
```

Comment:
Importing `Buffer` from `buffer` is actually a node.js module, is this intended? Because you installed a package called `buffer`  which specifically says that if you want to import the `Buffer` from that package you should import it as such:
```js
import { Buffer } from 'buffer/' // the '/' is needed to differentiate between node's 'buffer' module and the 'buffer' package
```
See:
https://github.com/feross/buffer#usage

Actually, I don't even think we need this import and the `buffer` package altogether since you provide `Buffer` with webpack:  https://github.com/instructure/instructure-ui/pull/913/files#diff-a728496508f687dee8416140d4a823a213effc80ff956540f63262e43db14ef7R43

https://webpack.js.org/plugins/provide-plugin/

Reply:
I removed the buffer import here, and it seems to be working fine.

Reply:
oh yeah, this was left here by accident, the provide plugin is enough in `uii-webpack-config/config/plugins.js`